### ISSUE TITLE
Blessing a client - QR code - process needs to be streamlined

### ISSUE THREAD

**Issue by bbusenius** [hahnicity/megthunderbird#4]
The QR code shouldn't have a button. Instead, this process should be automated. After the user takes a picture of the QR code with his or her phone, the symmetric key data should be saved. 

![qr-code](https://cloud.githubusercontent.com/assets/7826429/16474616/d0fb2e56-3e3d-11e6-9302-2e2d4c0a918a.png)

**Comment by thompsonm-ANL** [comment]
Also, if the scan doesn't work on the first try or your screen locks, there's no way to re-initiate the process.

**Comment by bbusenius** [comment]
Right, and on that note there's also no way of initiating it without sending an email to yourself or someone else. It should probably be opened as a separate issue but we need to think about how this dialog is launched in the first place.
